blueprints/foundational/README.md

```diff
@@ -0,0 +1,106 @@
+# AWS Landing Zone
+
+## Overview
+
+Contains the configuration for the [AWS Landing Zone Accelerator (LZA)](https://awslabs.github.io/landing-zone-accelerator-on-aws/latest/).
+It uses the [aws-luminarlz-cli](https://github.com/superluminar-io/aws-luminarlz-cli)
+to manage and deploy the LZA config.
+
+### Directory structure
+
+* [templates](templates) contains the liquid templates and other files that are used to generate the LZA config.
+* [customizations](customizations) contains a CDK app used to generate the Cloudformation templates that are defined in the [customizations-config](templates/customizations-config.yaml.liquid).
+* [config.ts](config.ts) defines how the templates are generated and all project relevant configurations.
+* [docs](docs) contains:
+  * [Architecture Decision Records (ADRs)](docs/adrs) that document project-specific decisions.
+  * [Runbooks](docs/runbooks) that help you with Landing Zone tasks.
+  * [Logs](docs/logs) a history of manual steps as part of the Landing Zone setup.
+
+### Customizations
+
+Part of the landing zone are [customizations](https://docs.aws.amazon.com/solutions/latest/landing-zone-accelerator-on-aws/customizing-the-solution.html)
+that are deployed via the LZA.
+
+These are configured in the [customizations-config](templates/customizations-config.yaml.liquid) file.
+LZA supports deploying Cloudformation templates
+which in this project are generated via the [customizations](customizations/bin/customizations.ts) CDK app.
+
+## Development
+
+### Prerequisites
+
+* Credentials for the `Management` AWS account.
+* Install the node version defined in [.node-version](.node-version).
+* Install dependencies:
+```bash
+npm i
+```
+
+### Manual synth of the LZA config
+
+```bash
+npm run cli -- synth
+```
+
+### Validate the LZA config
+
+This allows you to validate the LZA config before deploying it.
+Underneath it uses the [LZA Core CLI](https://awslabs.github.io/landing-zone-accelerator-on-aws/latest/developer-guide/scripts/#core-cli).
+
+```bash
+npm run cli -- lza config validate
+```
+
+### Deploy a LZA customizations stack
+
+This allows you to deploy a LZA customizations stack manually during development.
+Underneath it uses the [LZA Core CLI](https://awslabs.github.io/landing-zone-accelerator-on-aws/latest/developer-guide/scripts/#core-cli).
+
+```bash
+# synth
+npm run cli -- lza customizations-stack synth --stack-name <target-stack-name> --account-id <target-account-id>
+# e.g.: npm run cli -- lza customizations-stack synth --stack-name LzaCustomization-AwsAcceleratorPipeline --account-id <<AWS_MANAGEMENT_ACCOUNT_ID>>
+
+# synth & deploy
+npm run cli -- lza customizations-stack deploy --stack-name <target-stack-name> --account-id <target-account-id>
+```
+
+### Deploy a LZA pipeline stage
+
+This allows you to deploy a specific stage of the LZA pipeline for development or debugging purposes.
```

Comment:
I think I don't really understand this. Probably I just didn't read all the LZA docs. Since I would not necessarily assume that not every user will be LZA-savy, it might be a good idea to go into a bit more details here. Also an (additional) example that does not use defaults would be nice, I think.